src/boot/efi/boot.c

```diff
@@ -97,6 +97,10 @@ typedef struct {
         bool beep;
         int64_t console_mode;
         int64_t console_mode_efivar;
+        uint8_t color_normal;
+        uint8_t color_entry;
+        uint8_t color_highlight;
+        uint8_t color_edit;
```

Comment:
i hate this, but if people really want this i might be ok with this. but these should not be individual fields, as that just causes too much code duplication.

i.e. define an enum that lists all relevant colors.

then make this a `uint8_t colors[_COLOR_MAX]` array or so.